src/components/notificationPopUp/NotificationPopUp.tsx

```diff
@@ -54,12 +57,15 @@ export const NotificationPopUp: React.FC<NotificationPopUpProps> = ({
         <div className={styles.buttons}>
           {secondaryButton && (
             <div onClick={() => handleClick(secondaryButton.handleClick)}>
-              <Link icon={<CloseIcon />} iconAlign="start">
+              <Link icon={secondaryButton.icon ? secondaryButton.icon : <CloseIcon />} iconAlign="start">
```

Comment:
I think we could get rid of this ternary expression within the return template by setting the `secondaryButton.icon` beforehand, like so (dummy code).

_Before your return statement_

`secondaryButton.icon = secondaryButton.icon ?? <CloseIcon />`